.github/workflows/build-and-deploy.yml

```diff
@@ -52,6 +52,10 @@ jobs:
           npm install
       - name: Run unit tests
         run: npm run test
+        env:
+          JWT_ACCESS_SECRET: ${{secrets.JWT_ACCESS_SECRET}}
+          JWT_REFRESH_SECRET: ${{secrets.JWT_REFRESH_SECRET}}
+          DATABASE_URL: ${{secrets.DATABASE_URL}}
```

Comment:
Nice! Can see you put the secrets on Github as well. Thanks for this.

Reply:
Yup, I hope you guys don't mind it as env: expression for now. Please, feel free to modify the way the private variables are passed in the pipeline, as I'm not familiar the strategies you folks are using for CI/CD.